draft-ietf-tls-tls13.md

```diff
@@ -3554,13 +3571,6 @@ This document uses several registries that were originally created in
 {{RFC4346}}. IANA has updated these to reference this document. The registries
 and their allocation policies (unchanged from {{RFC4346}}) are listed below.
 
--  TLS ClientCertificateType Identifiers Registry: Future values in
-  the range 0-63 (decimal) inclusive are assigned via Standards
-  Action {{RFC2434}}.  Values in the range 64-223 (decimal) inclusive
-  are assigned via Specification Required {{RFC2434}}.  Values from
-  224-255 (decimal) inclusive are reserved for Private Use
-  {{RFC2434}}.
-
```

Comment:
Can we retire a registry that is still in use? It’s just the TLS 1.3 spec that no longer uses it (assuming the PR gets integrated). I’m inclined to let the editor make this change if he thinks it’s needed.

From: Dave Garrett [mailto:notifications@github.com]
Sent: Wednesday, October 14, 2015 2:35 PM
To: tlswg/tls13-spec <tls13-spec@noreply.github.com>
Cc: Andrei Popov <Andrei.Popov@microsoft.com>
Subject: Re: [tls13-spec] Removing certificate_types and adding certificate_extensions (#290)


In draft-ietf-tls-tls13.md<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2ftlswg%2ftls13-spec%2fpull%2f290%23discussion_r42056737&data=01%7c01%7cAndrei.Popov%40microsoft.com%7c8b66d4d48b9a4bc6ceeb08d2d4df5f48%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=hHo5aA9fftSLb%2f52tstZMumaCmIGPXtItBNzmJB8XPA%3d>:

> @@ -3554,13 +3571,6 @@ This document uses several registries that were originally created in

>  {{RFC4346}}. IANA has updated these to reference this document. The registries

>  and their allocation policies (unchanged from {{RFC4346}}) are listed below.

>

> --  TLS ClientCertificateType Identifiers Registry: Future values in

> -  the range 0-63 (decimal) inclusive are assigned via Standards

> -  Action {{RFC2434}}.  Values in the range 64-223 (decimal) inclusive

> -  are assigned via Specification Required {{RFC2434}}.  Values from

> -  224-255 (decimal) inclusive are reserved for Private Use

> -  {{RFC2434}}.

> -

The IANA Considerations section should probably declare retirement of the ClientCertificateType registry.

—
Reply to this email directly or view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2ftlswg%2ftls13-spec%2fpull%2f290%2ffiles%23r42056737&data=01%7c01%7cAndrei.Popov%40microsoft.com%7c8b66d4d48b9a4bc6ceeb08d2d4df5f48%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=lEC%2bhe3ctdWIb2DrNbDaQxvOsDi%2bm%2fV9VLkMINrYv4o%3d>.